### PR TITLE
Bug fix cortex xdr endpoints

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23080

## Description
Add the option to run xdr-get-endpoints command  with no filters and get all the endpoints back

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml [hunk 1]
```diff
@@ -1402,7 +1403,1400 @@ script:
   longRunning: false
   longRunningPort: false
   runonce: false
-  script: '-'
+  script: |-
```
Comment (ronykoz) [inline Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml:1406]: remove the script from here

Comment (ronykoz) [reply Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml:1406]: this is the unified file, please upload the yml alone

Comment (barchen1) [reply Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml]: done

**Comment by barchen1** [comment]
@ronykoz there is already a test which using the filters

**Pull request merged**